ember-client/src/main/scala/org/http4s/ember/client/EmberClientBuilder.scala

```diff
@@ -40,40 +40,12 @@ final class EmberClientBuilder[F[_]: Concurrent: Timer: ContextShift] private (
     private val logger: Logger[F],
     val chunkSize: Int,
     val maxResponseHeaderSize: Int,
+    private val idleConnectionTime: Duration,
```

Comment:
We've never been consistent about what's a public val on a builder and what isn't.

Reply:
We have not. We'd need to be consistent in our configurations. I just worry that I still have my timeouts wrong and wanted to give myself wiggle room in case I screwed up.